boards/arm/stm32_min_dev/Kconfig.defconfig

```diff
@@ -17,4 +17,14 @@ config UART_STM32_PORT_1
 
 endif # UART_CONSOLE
 
+if SPI
+
```

Comment:
Did you try to enable interrupt mode?
```
config SPI_STM32_INTERRUPT
	default y
```

Reply:
Yes, have tried this too but that did not help.